AppConnectSwift.xcodeproj/project.pbxproj

```diff
@@ -251,7 +254,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods-AppConnectSwift/Pods-AppConnectSwift-resources.sh\"\n";
+			shellScript = "\"${SRCROOT}/Pods/Target Support Files/Pods/Pods-resources.sh\"\n";
```

Comment:
We have this fork created before the fix for physical devices which were merged to develop. I think we need to have these changes in develop too which kind of adds to https://github.com/mdsol/appconnect-ios/pull/25